### PR TITLE
add CODEOWNERS

### PR DESCRIPTION
**Pull request by finn-tbd**
This repo is a collection of TBD examples from across our github. Most of them didn't have codeowners, so I've had to guess who was a good contact for each one. If you're listed, please let me know if you are a good codeowner for that example and if there should be any additional codeowners for it. I'll merge when I get confirmation from everyone.

* [x] `javascript/book-reviews` @acekyd @angiejones
* [x] `javascript/dinger` @blackgirlbytes - note this is the code [from the examples folder in developer.tbd.website](https://github.com/TBD54566975/developer.tbd.website/tree/main/examples/tutorials/dinger-completed), _not_ [TBD54566975/dinger](https://github.com/TBD54566975/dinger).
* [x] `javascript/dinger-starter` @blackgirlbytes 
* [x] `javascript/fan-club-credential` @nitro-neal
* [x] `javascript/kcc-prototype-exemplar` @KendallWeihe
* [x] `javascript/pfi-aud-usd-tbdex` @michaelneale
* [x] `javascript/shared-todo` @acekyd 
* [x] `javascript/shared-todo-starter` @acekyd 
* [x] `javascript/tbdex-pfi-exemplar` @michaelneale 
* [x] `javascript/todo` @acekyd @blackgirlbytes
* [x] `javascript/todo-starter` @acekyd @blackgirlbytes
* [x] `kotlin/tbdex-example-android` @michaelneale 
* [x] `swift/tbdex-example-ios` - @kirahsapong

### CODE REVIEW

**Comment by michaelneale** [comment]
@finn-tbd I checked myself on them - but can we add a few others from the commit history?

**Comment by finn-tbd** [comment]
For sure, I'm not seeing a lot of other committers for most of the ones you're a on @michaelneale but `tbdex-pfi-exemplar` seems to actually have it's own CODEOWNERS file that I missed, so I've copied everyone from there into here. Also added @jiyoontbd to `tbdex-example-android`

**Review by jiyoontbd — APPROVED** [review]

**Comment by michaelneale** [comment]
at what point do we move examples to this repo vs maintain in others?

**Comment by finn-tbd** [comment]
Should be good to start doing that any time, I was going to start doing that next week. I'm planning to submit a PR to each of the other repos wiping out all files and replacing the README with a short message linking to the new location.

**Comment by blackgirlbytes** [comment]
I agree with this. I think you can add me as an additional code owner for javascript/fan-club-credential

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# any files outside of examples, and examples that don't yet have a codeowner
+*                                 @finn-tbd
+
+javascript/book-reviews           @acekyd
```
Comment (angiejones) [inline CODEOWNERS:4]:
```suggestion
javascript/book-reviews           @acekyd @angiejones 
```

**Review by angiejones — APPROVED** [review]

**Comment by acekyd** [comment]
LGTM!

**Comment by finn-tbd** [comment]
Thanks all!

**Pull request merged**